src/openms/source/ANALYSIS/QUANTITATION/ItraqConstants.cpp

```diff
@@ -209,7 +209,7 @@ namespace OpenMS
       info.description = "";
       info.name = channel_names[itraq_type].getValue(i, 0);
       info.id = (Int)i;
-      if (!reporter_mass_exact.has(info.name))
+      if (!(reporter_mass_exact.find(info.name) != reporter_mass_exact.end()))
```

Comment:
Please replace check for non-existence with the simpler:
if (reporter_mass_exact.find(info.name) == reporter_mass_exact.end())

Reply:
@matteopilz